libsolidity/ast/Types.cpp

```diff
@@ -173,7 +173,7 @@ void StorageOffsets::computeOffsets(TypePointers const& _types)
 			byteOffset = 0;
 		}
 		if (slotOffset >= bigint(1) << 256)
-			BOOST_THROW_EXCEPTION(Error(Error::Type::TypeError) << util::errinfo_comment("Object too large for storage."));
+			BOOST_THROW_EXCEPTION(Error(8708_error, Error::Type::TypeError) << util::errinfo_comment("Object too large for storage."));
```

Comment:
Perhaps we should report an error in a regular way. I plan to do it next. Then, hopefully, we'll have no more cases where `Error` is thrown.

Reply:
I don't think it is possible here. I think we should use `solAssert` here.

Reply:
OK, changed. But I would like to have a closer look later.